### PR TITLE
Introduce common module for samples

### PR DESCRIPTION
**Pull request by grassydragon**
I've moved the common dependencies from the individual `build.gradle` files in the samples to the common module. Additionally, I've removed the Material library dependency from the `build.gradle` file in the `sceneview_1_0_0` module.

### CODE REVIEW

File: sceneview_1_0_0/build.gradle [hunk 1]
```diff
@@ -115,9 +115,6 @@ dependencies {
     api "androidx.fragment:fragment-ktx:1.5.5"
     api "androidx.lifecycle:lifecycle-runtime-ktx:2.5.1"
 
-    // Android Material
-    implementation 'com.google.android.material:material:1.7.0'
```
Comment (hshapley) [inline sceneview_1_0_0/build.gradle:119]: Shouldn't it be removed from the sceneview (pre-1.0.0) module as well?

Comment (ThomasGorisse) [reply sceneview_1_0_0/build.gradle:119]: You are right

Comment (grassydragon) [reply sceneview_1_0_0/build.gradle:119]: I'll try to remove it (there was a dependency in the `LoadingNode` class).

Comment (grassydragon) [reply sceneview_1_0_0/build.gradle:119]: I've removed the remaining dependencies and the `LoadingNode` class.

File: settings.gradle [hunk 2]
```diff
@@ -12,11 +12,8 @@ include ':arsceneview'
...
-include ':sample-ar-model-viewer'
-project(':sample-ar-model-viewer').projectDir = new File('samples/ar-model-viewer')
-include ':sample-model-viewer'
-project(':sample-model-viewer').projectDir = new File('samples/model-viewer')
\ No newline at end of file
+include ':samples:common'
+include ':samples:ar-cloud-anchor'
+include ':samples:ar-cursor-placement'
+include ':samples:ar-model-viewer'
+include ':samples:model-viewer'
```
Comment (ThomasGorisse) [inline settings.gradle:19]: Nice found!

Comment (ThomasGorisse) [inline settings.gradle:19]: Nice found!

**Review by ThomasGorisse — APPROVED** [review]
Great job @grassydragon and @hshapley !

**Pull request merged**